src/DotNetLightning.Core/Channel/Channel.fs

```diff
@@ -388,6 +388,32 @@ module Channel =
                 [] |> Ok
 
         // ---------- normal operation ---------
+        | ChannelState.Normal state, GeewalletPayment cmd when state.LocalShutdown.IsSome || state.RemoteShutdown.IsSome ->
```

Comment:
please rename all occurrences of GeewalletPayment to MonoHopUnidirectionalPayment; I'd prefer DNL to not mention geewallet at all